modin/pandas/base.py

```diff
@@ -1527,10 +1527,14 @@ def get(self, key, default=None):  # noqa: PR01, RT01, D200
         """
         Get item from object for given key.
         """
-        if key in self.keys():
-            return self.__getitem__(key)
-        else:
-            return default
+
+        def _get(key, list_like):
```

Comment:
I'd rather not have this inner function if we can avoid it. Why not use the try-except that pandas uses?

Reply:
There was a CI test failure deep in `take_2d_positional` for a different exception when I just passed in a non-list key.

Reply:
No - it was only our implementation.

Reply:
Waiting on implementation for: https://github.com/modin-project/modin/issues/5042

Reply:
Changes should be made now.